action.yaml

```diff
@@ -18,5 +18,5 @@ runs:
       run: pip install -e "${{ github.action_path }}"
       shell: bash
     - name: Run validator
-      run: python3 -m "aws_doc_sdk_examples_tools.validate" --root "${{ inputs.root }}" --doc_gen_only "${{ inputs.doc_gen_only }}"
+      run: python3 -m "aws_doc_sdk_examples_tools.validate" --root "${{ inputs.root }}" --doc_gen_only "${{ inputs.doc_gen_only }}" --strict-titles "${{ inputs.strict_titles }}"
```

Comment:
You need to add `strict-titles` to the inputs, above

Reply:
Missed that. Fixed.